client/app/locales/locale-de.json

```diff
@@ -0,0 +1,10 @@
+{
+  "LOGIN": {
+    "TITLE": "Login",
+    "EMAIL": "E-Mail",
+    "PASSWORD": "Passwort",
+    "WRONG": "Die Anmeldedaten sind fehlerhaft. Bitte überprüfe sie sorgfältig.",
+    "SUBMIT": "Login",
+    "SIGNUP": "Neuanmeldung"
+  }
+}
```

Comment:
I will think about including `webpack-angular-translate` later, not in this issue. Would still use `translate-default`, it could make the templates much easier to understand. Include the english string there.